### ISSUE TITLE
[DevTools] Compatibility with translatable models ? 

### ISSUE THREAD

**Issue by YannikFirre** [Laravel-Backpack/ideas#228]

**Comment by tabacitu** [comment]
Yup, I agree. Being able to make models translatable from T0 would be awesome. There are a number of ways we could do this... the easiest one I can think of from a UX perspective is that we add a `translatable` column type... If that's used even once... that model becomes translatable.

**Comment by tabacitu** [comment]
Will do @YannikFirre , thanks. Heads-up though - I've scheduled this for DevTools v2 - so we'll start working on it in March 2023. In the meantime we're hard at work getting Backpack's core to zero-issues and building Backpack v6 🎉
